src/azul/service/manifest_service.py

```diff
@@ -801,7 +801,7 @@ def _create_pipeline(self):
     def _hit_to_doc(self, hit: Hit) -> MutableJSON:
         return self.service.translate_fields(self.catalog, hit.to_dict(), forward=False)
 
-    column_joiner = ' || '
+    padded_column_joiner = f' {config.manifest_column_joiner} '
```

Comment:
Qualifying the name with "padded" misleadingly suggests that there is an "unpadded" column joiner. The `||` string in metadata is not a column joiner, since it's not used in columns. Columns are specific to tabular data. HCA metadata is not tabular. It's JSON.